src/pages/products/testing-center/index.tsx

```diff
@@ -0,0 +1,181 @@
+            <ValuePropCard title="The most secure exams">
+              Maximize test integrity with managed computers, lab firewall, and
+              vigilant proctoring. Students access only approved resources,
+              while cheating risks are minimized through computer screens, video
+              cameras and professional oversight.
+            </ValuePropCard>
+            <ValuePropCard title="Restriction to unauthorized AI services">
+              Ensure exam integrity by restricting access to AI services and
+              question-answering websites (such as chatGPT, Copilot, Chegg, and
+              CourseHero), fostering academic honesty and fair assessment.
+            </ValuePropCard>
+            <ValuePropCard title="Professional proctoring staff">
+              Ensure a fair and equitable experience for all students by using
+              professional proctors who are trained to uphold the testing center
+              policies and prevent violations. In addition, this relieves
+              instructors from proctoring duties, freeing course staff time for
+              other essential activities.
+            </ValuePropCard>
+            <ValuePropCard title="Reliable check-in process">
+              Enhance security measures by verifying students&apos; identities
+              at the lab entrance through their student ID cards. In addition,
+              all personal belongings, including phones, are securely stored
+              before entering the testing environment.
+            </ValuePropCard>
+            <ValuePropCard title="Flexible and efficient scheduling">
+              Offer exams asynchronously over 2-3 days, optimizing space
+              utilization in smaller computer labs for larger classes. Eliminate
+              the need for instructors to manage make-up or conflict exams,
+              streamlining the testing process for both students and faculty.
+            </ValuePropCard>
+            <ValuePropCard title="Student-centric exam scheduling">
+              Empower students, especially those balancing family and work
+              obligations, with the flexibility to schedule exams at their
+              preferred times within the exam period. Enable easy reservation
+              modifications and cancellations up to the selected exam time,
+              providing convenience and autonomy.
+            </ValuePropCard>
+            <ValuePropCard title="Seamless exam overrides">
+              Empower instructors with the capability to effortlessly grant
+              overrides for students facing unexpected circumstances, such as
+              illness or unavoidable absences during the exam period.
+            </ValuePropCard>
+            <ValuePropCard title="Authentic testing">
+              Create an authentic testing environment by providing students with
+              access to computers equipped with compilers, debuggers, and
```

Comment:
Nit (in our desired configuration with workspaces, the computers in the testing center don't actually have any of these, but regardless, I think this is a more concise way to say the same thing)

```suggestion
              access to compilers, debuggers, and
```